acpi.adoc

```diff
@@ -38,10 +38,6 @@ software-defined interfaces and the complexity in supporting ACPI is reduced.
 The following additional BRS requirements apply:
 
 * No FACS table.
-* Both interrupt-signaled and GPIO-signaled ACPI Events are supported
-  (cite:[ACPI] Section 5.6)
-* There is at least 1 wake event.
```

Comment:
Not in the ACPI spec either.

Reply:
It is implied by the fact that the system is able to be turned on.

Reply:
Is it wake from off or suspend? If it's off, yes, it's implied a system can be turned on. If the latter that's a different issue.

That said, Andrei, what are you wanting to obtain through this requirement?

Reply:
Ok, we can drop the "at least 1 wake event" part.